packages/@aws-cdk/aws-lambda/lib/function-base.ts

```diff
@@ -537,6 +562,10 @@ export abstract class QualifiedFunctionBase extends FunctionBase {
       ...options,
     });
   }
+
+  public addPermission(id: string, permission: Permission): void {
+    super.addPermissionHelper(id, permission, true);
```

Comment:
Do all of the Qualified functions actually use this base class? Looks like 'LatestVersion' does not.

Reply:
Did some research on this, but will update it on the new PR i've created.